### ISSUE TITLE
This seems not to be working with WEG 1.4.6

### ISSUE THREAD

**Issue by Ubsefor** [hieplpvip/AppleBacklightSmoother#12]
Can not change display brightness at all when I update to WEG 1.4.6, (switching the boot-args included, as stated in the instruction) not sure whether it is AppleBacklightSmoother problem or WEG’s (or even OpenCore’s), but reporting the issue as soon as discovered. Will test other combinations of this extension with updated versions of Acidanthera kexts later.

**Comment by Ubsefor** [comment]
It turns out, that WEG doesn’t patch backlight registers by default, so we do not need `igfxcflbklt=0` or it’s newer equivalent `-igfxblr` for this kext to work correctly. Suggest removing this from installation instructions.

**Comment by Ubsefor** [comment]
More compatibility problems with WEG ≥ 1.4.6 are described in Acidanthera’s bugtracker [#1471](https://github.com/acidanthera/bugtracker/issues/1471)

TL;DR:
The kext causes recovery environment to KP due to page fault.

**Comment by antoniomcr96** [comment]
I had to disable the kext as it probably caused me problems entering recovery mode. I disabled some kext, but after reading this issue I suppose this could be the problem. Hope @hieplpvip can fix the issue soon

**Comment by tylernguyen** [comment]
@hieplpvip 

I've confirmed this issue also, please take a look.

**Comment by LeeBinder** [comment]
@Ubsefor you have written 1.**5**.6 THREE times.. but even the link you post points to WEG 1.**4**.6..... and even now, WEG is at v. 1.4.x. Please correct the misleading issue title - thanks.

@tylernguyen @antoniomcr96 what about your eye's vision and alertness?

**Comment by Ubsefor** [comment]
> @Ubsefor you have written 1.**5**.6 THREE times.. but even the link you post points to WEG 1.**4**.6..... and even now, WEG is at v. 1.4.x. Please correct the misleading issue title - thanks.
> 
> @tylernguyen @antoniomcr96 what about your eye's vision and alertness?

Not much of an issue, as everyone else understood me 🙂

Fixed it anyway, thanks

**Comment by LeeBinder** [comment]
mighty fine - you da man ;)
